login-workflow/src/screens/subScreens/ExistingAccountComplete.tsx

```diff
@@ -13,7 +13,7 @@ export const ExistingAccountComplete: React.FC<React.PropsWithChildren<React.Pro
     const { t } = useLanguageLocale();
 
     return (
-        <FinishState
+        <FinishState            
```

Comment:
it appears that you still did not run `yarn prettier`

Reply:
I have added Prettier extension to my VS code as per your earlier comment and but missed running `yarn prettier`.